apps/nextjs-website/src/lib/webinars/__tests__/webinarSubscriptions.test.ts

```diff
@@ -0,0 +1,115 @@
+import { mock } from 'jest-mock-extended';
+import * as E from 'fp-ts/lib/Either';
+import {
+  WebinarEnv,
+  WebinarSubscription,
+  deleteWebinarSubscription,
+  insertWebinarSubscription,
+  listUserWebinarSubscriptions,
+} from '../webinarSubscriptions';
+import {
+  DeleteItemCommand,
+  PutItemCommand,
+  QueryCommand,
+} from '@aws-sdk/client-dynamodb';
+import { makeDynamodbItemFromWebinarSubscription } from '../dynamodb/codec';
+
+const aWebinarSubscription: WebinarSubscription = {
+  username: 'aUsername@mail.com',
+  webinarId: 'a-webinar-id',
+};
+const aInsertWebinarSubscription: WebinarSubscription = {
+  username: aWebinarSubscription.username,
+  webinarId: aWebinarSubscription.webinarId,
+};
+const aDynamoDBItem = makeDynamodbItemFromWebinarSubscription({
+  ...aWebinarSubscription,
+});
+
+const makeTestWebinarEnv = () => {
+  const dynamoDBClientMock = mock<WebinarEnv['dynamoDBClient']>();
+  // default mock behaviour
+  dynamoDBClientMock.send.mockImplementation(async (cmd) => {
+    if (cmd instanceof PutItemCommand) return {};
+    else if (cmd instanceof DeleteItemCommand) return {};
+    else if (cmd instanceof QueryCommand) return { Items: [aDynamoDBItem] };
+    // eslint-disable-next-line functional/no-throw-statements
+    else throw new Error('Unsupported command');
+  });
+  const env = {
+    dynamoDBClient: dynamoDBClientMock,
+  };
+  return { env, dynamoDBClientMock };
+};
+
+describe('webinarSubscriptions', () => {
+  describe('insertWebinarSubscription', () => {
+    it('should send dynamodb put command', async () => {
+      const { env, dynamoDBClientMock } = makeTestWebinarEnv();
+      const actual = await insertWebinarSubscription(
+        aInsertWebinarSubscription.webinarId,
+        aInsertWebinarSubscription.username
+      )(env)();
+      const expected = E.right(undefined);
+
+      expect(dynamoDBClientMock.send).toBeCalledTimes(1);
+      expect(actual).toStrictEqual(expected);
+    });
+    it('should return error if send returns an error', async () => {
```

Comment:
Nice job to add a test to check error case here :+1:  But why this test is only here for adding a subscription and there are not similar test for delete subscription and get subscriptions cases ' :thinking: